### PR TITLE
Add `count` to `TagSerializer`

### PR DESCRIPTION
**Pull request by jerabekjiri**
fixes: #1612

### CODE REVIEW

**Review by mdellweg — APPROVED** [review]

**Comment by gerrod3** [comment]
I would suggest testing this change first on a large system. You are adding N db queries to each tag list call with this change.

**Comment by mdellweg** [comment]
> I would suggest testing this change first on a large system. You are adding N db queries to each tag list call with this change.

Right. Can we use some annotation logic?

File: pulp_ansible/app/serializers.py [hunk 1]
```diff
@@ -385,9 +385,14 @@ class TagSerializer(serializers.ModelSerializer):
     A serializer for the Tag model.
     """
 
+    count = serializers.SerializerMethodField()
+
+    def get_count(self, obj):
+        return obj.count
```
Comment (mdellweg) [inline pulp_ansible/app/serializers.py:391]: I think it knows how to do that now... (Please try removing it).

You need to change the SerializerMethodField for something else...

Comment (jerabekjiri) [reply pulp_ansible/app/serializers.py]: changed to integer field

**Pull request merged**